docs/README.rst

```diff
@@ -6,7 +6,7 @@ This allows it to be built into other forms for easier viewing and browsing.
 
 To create an HTML version of the docs:
 
-* Install Sphinx (using ``python -m pip install Sphinx`` or some other method).
+* From the project root directory run ``python -m pip install -r docs/requirements.txt``
 
 * In this docs/ directory, type ``make html`` (or ``make.bat html`` on
```

Comment:
Given this sentence is sort of assuming the user is in the `docs/`folder, perhas the previous modified paragraph could suggest to `cd` into that folder and install `requirements.txt` from there?